src/utilities.cpp

```diff
@@ -136,7 +136,8 @@ void renameFile(FmFileInfo *file, QWidget *parent) {
 
   GFile* gf = fm_path_to_gfile(path);
   GFile* parent_gf = g_file_get_parent(gf);
-  GFile* dest = g_file_get_child(G_FILE(parent_gf), new_name.toLocal8Bit().data());
```

Comment:
@palinek, I also think there was no problem in the first place at all. However, I think @luis-pereira's PR is *logically* good because `g_file_get_child()` gets a const char*.